Nautilus/Patchers/PDAEncyclopediaPatcher.cs

```diff
@@ -1,5 +1,6 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using HarmonyLib;
+using Nautilus.Handlers;
```

Comment:
does this import need to be added?

Reply:
No, this was leftover from previous modifications to that file that I removed, seems I forgot to remove the import.

Reply:
Fixed.